Resources/Prototypes/Nyanotrasen/Markers/Spawners/untimedAISpawners.yml

```diff
@@ -49,8 +101,10 @@
         - MobXeno
         - MobXenoPraetorianNPC
         - MobXenoDroneNPC
-        - MobXenoQueenNPC
         - MobXenoRavagerNPC
         - MobXenoRunnerNPC
         - MobXenoRounyNPC
         - MobXenoSpitterNPC
+      rarePrototypes:
+        - MobXenoQueenNPC
+      rareChance: 0.10
```

Comment:
this make xeno queen a 10% chance which is about what it was before lol

Reply:
Yeah, I guess, haha, I just wanted to move it down to rarechance so it could be tweaked easier if needed.